hashintel/hash
# Final Header Fixes

## 🌟 What is the purpose of this PR?




This PR fixes over the final set of changes described in #375. Although further changes might pop up after this one, this one encompasses the following changes:
1. [Display a user's primary email](https://github.com/hashintel/hash/pull/397/files#diff-795901c71d9a5e5302bdb94610f90971b7b5e2f971fcde694d1d688338b3b561R33) in the account header.
2. [Destructure the theme object](https://github.com/hashintel/hash/pull/397/files#diff-795901c71d9a5e5302bdb94610f90971b7b5e2f971fcde694d1d688338b3b561R60) instead of using `useTheme`.
3. [Use LinkButton](https://github.com/hashintel/hash/pull/397/files#diff-158ee6a4d1ea1f40f46d8b1bc40a1230746af425e803a59f1282c41e88fb81f5R82) over buttons with onClick routing.

## 🛡 What tests cover this?



- Existing linting tools cover the correctness of code in this PR.

## ❓ How to test this?



1. Checkout the branch locally.
4. Click through the Page Header.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201925844970654
